StripePayments/StripePayments/Source/Helpers/STPBankAccountCollector.swift

```diff
@@ -459,4 +459,42 @@ public class STPBankAccountCollector: NSObject {
             completion(setupIntent, nil)
         }
     }
+
+    // MARK: - Collect Bank Account - Deferred Intent
+    @_spi(STP) public func collectBankAccountForDeferredIntent(
+        sessionId: String,
+        returnURL: String?,
+        onBehalfOf: String?,
+        from viewController: UIViewController,
+        financialConnectionsCompletion: @escaping (
+            FinancialConnectionsSDKResult?, LinkAccountSession?, NSError?
+        ) -> Void
+    ) {
+        guard
+            let financialConnectionsAPI = FinancialConnectionsSDKAvailability.financialConnections()
+        else {
+            assertionFailure("FinancialConnections SDK has not been linked into your project")
```

Comment:
Does it make sense to add a link to the docs on how they can fix this?

Reply:
Yeah although this part of the code won't get hit in practice, ach will get filtered out if the SDK isn't linked.  I'll throw a link into the relevant part of the code.